devices/CharacterLcd/custom/ImageSharp.cs

```diff
@@ -0,0 +1,71 @@
+﻿// This is a class added speifically for nanoFramework so we dont need to include the full fat ImageSharp lib!
+
+using System;
+
+namespace SixLabors.ImageSharp
+{
+    /// <summary>
+    /// Stores an ordered pair of integers, which specify a height and width.
+    /// </summary>
+    /// <remarks>
+    /// This struct is fully mutable. This is done (against the guidelines) for the sake of performance,
+    /// as it avoids the need to create new values for modification operations.
+    /// </remarks>
+    public struct Size
+    {
+        /// <summary>
+        /// Gets or sets the height of this <see cref="Size"/>.
+        /// </summary>
+        public int Height { get; set; }
+
+        /// <summary>
+        /// Gets or sets the width of this <see cref="Size"/>.
+        /// </summary>
+        public int Width { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Size"/> struct.
+        /// </summary>
+        /// <param name="width">The width of the size.</param>
+        /// <param name="height">The height of the size.</param>
+        public Size(int width, int height)
+        {
+            Width = width;
+            Height = height;
+        }
+    }
+
+    ///// <content>
```

Comment:
as mentioned, better to use the System.Drawing nuget we have for colors.

Reply:
I have added the latest `System.Drawing` nuget to aid this.